public/script/register_script.js

```diff
@@ -0,0 +1,35 @@
+// By default hide OTP input form.
+$('#otp-input-box').hide();
+
+// Register form submit.
+$(document).on('submit', '#form-user-data-for-register', function submitUserData(e) {
+  e.preventDefault();
+  // Password & reenter-password not matched.
+  if ($("input[name='password']").val() !== $("input[name='reenter-password']").val()) {
```

Comment:
This is done in multiple files.
Can we create a function for this for reusability ?